### ISSUE TITLE
piping an empty scss file into gulp-sass causes gulp to crash

### ISSUE THREAD

**Issue by alvint** [dlmanning/gulp-sass#155]
If one of your scss files is empty (0 bytes), and the file gets piped to gulp-sass, gulp crashes and produces this unhelpful error message:

```
after 461 ms
gulp(6075,0x7fff7a12d300) malloc: *** error for object 0x1048abe44: pointer being freed was not allocated
*** set a breakpoint in malloc_error_break to debug
Abort trap: 6
```

Normally this wouldn't be too big a deal, but since the error message gives no indication of where the error resides, this should be fixed to prevent developers spending an undue amount of time trying to find the source of the issue.

This issue may be specific to gulp 4 (I haven't tested with gulp 3).

**Comment by MethodGrab** [comment]
It's a `libsass` issue; it should be fixed in `gulp-sass >= 1.3.1` which uses `node-sass@2.0.0-beta`. More info in https://github.com/dlmanning/gulp-sass/issues/104 & https://github.com/sass/node-sass/issues/554

**Comment by alvint** [comment]
Yep, works here.
